x/exomint/keeper/params.go

```diff
@@ -0,0 +1,60 @@
+package keeper
+
+import (
+	"cosmossdk.io/math"
+	"github.com/ExocoreNetwork/exocore/x/exomint/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// GetMintDenom gets the mint denomination.
+func (k Keeper) GetMintDenom(ctx sdk.Context) string {
+	var mintDenom string
+	k.paramstore.Get(ctx, types.KeyMintDenom, &mintDenom)
```

Comment:
x/params has been depreacted ,and for the gov module to handle params update through proposal can just by setting the authority with govModule(it has registered the msgServiceRouter with all modules in its keeper). Maybe consider just use module store to house the params ?

Reply:
Done, thank you